projects/collection-editor-library/src/lib/interfaces/editor.ts

```diff
@@ -42,6 +42,7 @@ export interface Context {
     collectionObjectType?: string;
     collectionPrimaryCategory?: string;
     targetFWIds?: string[];
+    cloudStorage?: any;
```

Comment:
Where are you using this "cloudStorage"? 
This is not an interface property. This is the configuration property of the editor